parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -45,10 +45,6 @@ fi
 
 configureAdminUser
 
-{{- if not NeedsContainerd}}
-cleanUpContainerd
-{{end}}
-
```

Comment:
I think we still need this for the previous VHDs with  containerd and dockerd

Reply:
added back in